docker-compose.yml

```diff
@@ -1,6 +1,20 @@
 version: '3.8'
 
 services:
+  app:
+    build:
+      context: .
+      dockerfile: ./.docker/app/Dockerfile
```

Comment:
Why have a separate `.docker/` folder? Wouldn't it be simpler if it was just in the root of this repo? I can understand you want to separate the Dockerfile related files/project into its own directory, but I like simplicity and I also question if we truly need any files such as `php.ini` to be commited in this repository if such files could just be initialized and populated with env variable values on container startup.

Reply:
Actually I kind of like the idea of this .docker folder, after all. Nevermind my comment, keep it.